.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on: [push]
```

Comment:
Will this run on pull requests as well?

Reply:
If we change it to `on: [push, pull_request]` it will do what you ask

Reply:
I pushed a new version that adds this.  It is just "tacked on" to this current PR